### PR TITLE
systemd were unable to start the service with " characters

### PR DESCRIPTION
**Pull request by jostmart**
I spent a couple of hours trying to get the exporter to start. Finally found out it was crashing due to the " characters.

### CODE REVIEW

**Comment by jostmart** [comment]
Could this be merged?

File: tasks/main.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@
 
 - name: curl haproxy-exporter metrics
   uri:
-    url: http://localhost:9101/metrics
+    url: http://{{ haproxy_exporter_listen_address }}:{{ haproxy_exporter_listen_port }}/metrics
```
Comment (nikosmeds) [inline tasks/main.yml:95]: We're missing a default value for `haproxy_exporter_listen_address`, but otherwise this LGTM.

**Comment by nikosmeds** [comment]
> Could this be merged?

Sorry about the slow reply, I'll need to update my settings since I never received notifications for this.

**Comment by jostmart** [comment]
@nikosmeds I've added a default for the haproxy_exporter_listen_address.

**Comment by nikosmeds** [comment]
Thanks @jostmart. I've switched orgs (and thus cannot merge), but perhaps @slam1987 can takeover from here. Hello @slam1987 👋

**Comment by slam1987** [comment]
> Thanks @jostmart. I've switched orgs (and thus cannot merge), but perhaps @slam1987 can takeover from here. Hello @slam1987 👋

Hey @nikosmeds!

Thanks for the update @jostmart.

**Pull request merged**